docs/components_page/page.py

```diff
@@ -94,6 +118,36 @@ def __init__(self, app):
                 HighlightedSource(collapse_source),
                 ApiDoc(component_metadata.get("src/components/Collapse.js")),
             ],
+            "layout": html.Div(
+                [
+                    layout_simple,
```

Comment:
I think this (and all the other `layout_`) should be wrapped in the `ExampleContainer` component. It imposes some margins etc.

Reply:
Good point, I've added those.